src/System.Private.CoreLib/src/System/AppContext/AppContext.cs

```diff
@@ -130,88 +127,19 @@ public static bool TryGetSwitch(string switchName, out bool isEnabled)
             if (switchName.Length == 0)
                 throw new ArgumentException(SR.Argument_EmptyName, nameof(switchName));
 
-            // By default, the switch is not enabled.
-            isEnabled = false;
+            if (s_switches == null)
+            {
+                isEnabled = false;
+                return false;
+            }
 
-            SwitchValueState switchValue;
-            lock (s_switchMap)
+            int switchValue;
+            lock (s_switches)
             {
-                if (s_switchMap.TryGetValue(switchName, out switchValue))
-                {
-                    // The value is in the dictionary. 
-                    // There are 3 cases here:
-                    // 1. The value of the switch is 'unknown'. This means that the switch name is not known to the system (either via defaults or checking overrides).
```

Comment:
All this is desktop-specific appcompat infrastucture that was not doing anything useful for CoreCLR.